db/migrate/20151435234635_add_container_definition_security_contexts.rb

```diff
@@ -0,0 +1,13 @@
+class AddContainerDefinitionSecurityContexts < ActiveRecord::Migration
+  def change
+    add_column    :container_definitions, :privileged, :boolean
+    add_column    :container_definitions, :run_as_user, :bigint
+    add_column    :container_definitions, :se_linux_user, :string
+    add_column    :container_definitions, :se_linux_role, :string
+    add_column    :container_definitions, :se_linux_type, :string
+    add_column    :container_definitions, :se_linux_level, :string
```

Comment:
Can you move these `se_linux` columns to a better home in a slightly more normalized table with a polymorphic relationship?  Maybe something like `security_contexts`.  Then, `container_defintions` could `has_one :security_context`.  And, other entities could also start depending on this (`system_services` for instance).

I'm not sure if the other columns (`privileged`, `run_as_user`, and `run_as_non_root` should also move, but maybe).